primitives/src/lib.rs

```diff
@@ -22,20 +22,3 @@ pub use extrinsics::*;
 
 pub mod extrinsic_params;
 pub mod extrinsics;
-
-/// The block number type used in this runtime.
-pub type BlockNumber = u64;
```

Comment:
Not needed anymore, because all these params should now be generic inside the api-client.